Classroom changes 272 2 (#325)
cisagov/ScubaGoggles: baselines/Google Classroom Minimum Viable Secure Configuration Baseline v0.2.md

Reviewer: Also don't forget to update the table of contents.

### Implementation
To configure the settings for Student Unenrollment:

#### GWS.CLASSROOM.5.1v0.2 Instructions
1.  Sign in to the [Google Admin Console](https://admin.google.com).
2.  Select **Apps** -\> **Additional Google Service** -\> **Classroom**.
3.  Select **Student unenrollment**.
4.  Select **Teachers Only**.
5.  Select **Save**.

## 5. Class Creation

> This section covers who has the ability to create classes.

Reviewer: I'd recommend adding all the details in the introduction section that the reader will need in order to understand the policy. What is a verified teacher? What is a pending teacher? How does the verification process work? For example:

```suggestion
The first time users sign in to Classroom, they self-identify as either a student or teacher. Users who identify as teachers will be marked as a pending teacher until an administrator verifies them. Google Classroom allows administrators to restrict class creation to only verified teachers.
```

### Policy

#### GWS.CLASSROOM.5.1v0.2

> Who can create classes SHALL be set to Verified teachers only.

Reviewer: ```suggestion
Who can create classes SHALL be set to verified teachers only.
```

> - _Rationale:_ Allowing pending teachers to create classes allows for potential unauthorized data creation which creates a security risk. By allowing only verified teachers to create classes mitigates the risk.

Reviewer: I agree that there's a risk here, but I don't think "unauthorized data creation" exactly covers it. If I impersonate a teacher and create a class and some users join it, can I, as a pending teacher, see the PII of those students? It looks like I'd be able to see those students' [names, photos, and email addresses](https://support.google.com/edu/classroom/answer/6386395?hl=en#:~:text=Who%20can%20see%20what%3F&text=Teachers%20in%20your%20classes%20can,view%20your%20name%20and%20photo.). If nothing else, me creating a class as an unverified teacher creates an unfounded trust relationship between me and any students that join my class that I might be able to exploit, e.g., for phishing. These are what I see as the true risks here.

```suggestion
- _Rationale:_ Allowing pending teachers to create classes potentially allows students to impersonate teachers and exploit the trusted relationship between teacher and student, e.g., to phish sensitive information from the students. Restricting class creation to verified teachers reduces this risk.
```

- _Last modified:_ June 21, 2024

- MITRE ATT&CK TTP Mapping
  - Pending

### Resources

-   [Verify teachers and set permissions](https://support.google.com/edu/classroom/answer/6071551?hl=en)

### Prerequisites

-   None

### Implementation

> To configure the settings for Student Unenrollment:

Reviewer: Looks like a copy/paste error. This section isn't about student unenrollment.

Author: Fixed.

3.  Select **Student unenrollment**.
4.  Select **Teachers Only**.
5.  Select **Save**.

## 5. Class Creation

The first time users sign in to Classroom, they self-identify as either a student or teacher. Users who identify as teachers will be marked as a pending teacher until an administrator verifies them. Google Classroom allows administrators to restrict class creation to only verified teachers.

### Policy

#### GWS.CLASSROOM.5.1v0.2

> Who can create classes SHALL be set to verified teachers only.

Reviewer: One final possible enhancement. The current policy wording heavily leverages the wording used in the UI "buttons," so it reads a little awkward as plain-English. If you do make this change, don't forget to also make the change in the drift rules.

```suggestion
Class creation SHALL be restricted to verified teachers only.
```
